### DISCUSSION TITLE
5.1 surround sound on windows

### DISCUSSION THREAD

**Discussion by Bonesley** [tidal-music/discussions#72] [Ideas & Feature requests]
There are many surround mixes out there and having the option to stream 5.1 audio on the windows application would be great.

**Comment by stefanmoro** [comment]
Hey @Bonesley,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to our existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
